spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1278,6 +1281,7 @@ public void close() {
     if (numReferenceOfSparkContext.decrementAndGet() == 0) {
       sc.stop();
       sc = null;
+      sparkSession = null;
       if (classServer != null) {
```

Comment:
`stop` should be called on sparkSession before `sc.stop()`
http://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.sql.SparkSession

(as of now this is ok since `sparkSession.stop()` simply calls `sc.stop()` but this could change)